code/src/Components/Gamepage.js

```diff
@@ -0,0 +1,77 @@
+// /////////////// IMPORTS //////////////////////// //
+
+// Import React and necessary Redux hooks and components
+
+import React from 'react';
+import { labyrinthMango, continueGame } from 'Reducers/labyrinth';
+import { useDispatch, useSelector } from 'react-redux';
+import { CoordsImageDisplay } from './CoordsImageDisplay';
+import { Loader } from './Loader';
+import { GameWrapper } from '../Styles/Globalstyles';
+import { DescriptionWrapper, DirectionWrapper, RestartBtn, DirectionBtn, GameEnd } from '../Styles/Gamepage.styled';
+
+// /////////////// COMPONENT //////////////////////// //
+
+// Create Gamepage component
+export const Gamepage = () => {
+  // Set up hooks for dispatch and store
+  const dispatch = useDispatch()
+  const description = useSelector((store) => store.labyrinthMango.description)
```

Comment:
I see you stored all values in separate properties in the store. We decided to store the current position into one single object, that we called currentPosition. That way, we only needed one useSelector to get that object that held all the information. If you want less code, that might be a possible way of doing it. :)